tests/test_content_smart_proxy.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+
+@pytest.fixture
+def content_smart_proxy(api, organization):
+    smart_proxies = api.resource('smart_proxies').call('index')
+    content_proxy = {}
+
+    for proxy in smart_proxies['results']:
+        for feature in proxy['features']:
+            if feature['name'] == 'Pulp Node':
+                content_proxy = proxy
+                break
+
+    content_proxy_id = content_proxy['id']
+
+    environments = api.resource('capsule_content').call('lifecycle_environments', {
+        'id': content_proxy['id']
+    })['results']
+
+    for environment in environments:
+        api.resource('capsule_content').call('remove_lifecycle_environment', {
+            'id': content_proxy_id,
+            'environment_id': environment['id']
+        })
+
+    library = api.resource('lifecycle_environments').call('index', {
+        'search': 'Library',
+        'organization_id': organization['id']
+    })['results'][0]
+
+    api.resource('capsule_content').call('add_lifecycle_environment', {
+        'id': content_proxy_id,
+        'environment_id': library['id']
+    })
+
+    return content_proxy
+
+
+@pytest.fixture
+def organization(api, entities):
+    return api.resource('organizations').call('index', {
+        'search': entities['organization_label']
+    })['results'][0]
```

Comment:
I wonder if we can use get an org by label (show action) but https://theforeman.org/api/2.3/apidoc/v2/organizations/show.html doesn't really clarify this. I know in some places we can use some name instead of just the DB ID.